lib/active_merchant/billing/gateways/card_stream.rb

```diff
@@ -283,6 +284,20 @@ def add_threeds_required(post, options)
         add_pair(post, :threeDSRequired, options[:threeds_required] || @threeds_required ? 'Y' : 'N')
       end
 
+      def add_global_threeds2_mpi_data(post, options)
+        return unless three_d_secure = options[:three_d_secure]
```

Comment:
💬  Comment:

Given that CardStream only supports XID we can assume that only 3DS v1 is accepted so maybe a check of 3ds version would be nice.